mod_lookup_identity.c

```diff
@@ -38,7 +38,7 @@
 #define DBUS_SSSD_IFACE_USERS "org.freedesktop.sssd.infopipe.Users"
 #define DBUS_SSSD_GET_USER_GROUPS_METHOD "GetUserGroups"
 #define DBUS_SSSD_GET_USER_ATTR_METHOD "GetUserAttr"
-#define DBUS_SSSD_FIND_BY_CERTIFICATE "FindByCertificate"
+#define DBUS_SSSD_FIND_BY_NAME_AND_CERTIFICATE "FindByNameAndCertificate"
```

Comment:
Makes sense, otherwise we would force users to have latest SSSD even when they don't need it.